man/src/network/create.md

```diff
@@ -5,59 +5,60 @@ network driver you can specify that `DRIVER` here also. If you don't specify the
 When you install Docker Engine it creates a `bridge` network automatically. This
 network corresponds to the `docker0` bridge that Engine has traditionally relied
 on. When you launch a new container with  `docker run` it automatically connects to
-this bridge network. You cannot remove this default bridge network but you can
+this bridge network. You cannot remove this default bridge network, but you can
 create new ones using the `network create` command.
 
-```bash
+```console
 $ docker network create -d bridge my-bridge-network
 ```
 
 Bridge networks are isolated networks on a single Engine installation. If you
 want to create a network that spans multiple Docker hosts each running an
-Engine, you must create an `overlay` network. Unlike `bridge` networks overlay
-networks require some pre-existing conditions before you can create one. These
-conditions are:
+Engine, you must enable Swarm mode, and create an `overlay` network. To read more
+about overlay networks with Swarm mode, see ["*use overlay networks*"](https://docs.docker.com/network/overlay/).
 
-* Access to a key-value store. Engine supports Consul, Etcd, and Zookeeper (Distributed store) key-value stores.
-* A cluster of hosts with connectivity to the key-value store.
-* A properly configured Engine `daemon` on each host in the cluster.
+Once you have enabled swarm mode, you can create a swarm-scoped overlay network:
 
-The `dockerd` options that support the `overlay` network are:
-
-* `--cluster-store`
-* `--cluster-store-opt`
-* `--cluster-advertise`
-
-To read more about these options and how to configure them, see ["*Get started
-with multi-host
-network*"](https://docs.docker.com/engine/userguide/networking/get-started-overlay/).
-
-It is also a good idea, though not required, that you install Docker Swarm on to
-manage the cluster that makes up your network. Swarm provides sophisticated
-discovery and server management that can assist your implementation.
+```console
+$ docker network create --scope=swarm --attachable -d overlay my-multihost-network
+```
 
-Once you have prepared the `overlay` network prerequisites you simply choose a
-Docker host in the cluster and issue the following to create the network:
+By default, swarm-scoped networks do not allow manually started containers to
+be attached. This restriction is added to prevent someone that has access to
+a non-manager node in the swarm cluster from running a container that is able
+to access the network stack of a swarm service.
 
-```bash
-$ docker network create -d overlay my-multihost-network
-```
+The `--attachable` option used in the example above disables this restriction,
+and allows for both swarm services and manually started containers to attach to
+the oerlay network.
 
 Network names must be unique. The Docker daemon attempts to identify naming
 conflicts but this is not guaranteed. It is the user's responsibility to avoid
 name conflicts.
 
+### Overlay network limitations
```

Comment:
Synced the man pages markdown with the "documentation" markdown, so some content was added in this commit as well